docs/modules/data-structures/pages/working-with-external-data.adoc

```diff
@@ -250,6 +250,57 @@ include::ROOT:example$/dds/map/PersonEntryStore.java[tag=personms]
 
 TIP: link:https://github.com/hazelcast/hazelcast-code-samples/tree/master/distributed-map/mapstore/src/main/java[See more MapStore/MapLoader code samples].
 
+== Offloading MapStore and MapLoader Operations
+
+Offloading `MapStore` and `MapLoader` operations allow these operations to work asynchronously, meaning they do not block each other.
+This provides faster executions of these operations, resulting in higher throughput of the whole cluster.
```

Comment:
```suggestion
This provides faster executions of other operations in cluster, since partition threads are not blocked by MapStore/MapLoader calls, resulting in higher throughput of the whole cluster.
```